Fix aclocal warning:
/usr/share/aclocal/smpeg.m4:13: warning: underquoted definition of AM_PATH_SMPEG
--- smpeg-0.4.4/smpeg.m4
+++ smpeg-0.4.4/smpeg.m4
@@ -12,3 +12,3 @@
 dnl
-AC_DEFUN(AM_PATH_SMPEG,
+AC_DEFUN([AM_PATH_SMPEG],
 [dnl
